src/styles/common/_colors.scss

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2021 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+// Dell Design Standard Color codes
+
+$primary-white: #FFFFFF;
+$gray-100: #F7F7F7;
+$gray-200: #EEEEEE;
+$gray-300: #E4E4E4;
+$gray-450:  #BBBBBB;
```

Comment:
450, 550 ?   go with 500, 600

Reply:
Followed naming convention inline with DDS, I have only included the colors which are currently used https://web.ddsproduct.com/31b3fd8b1/p/60290f-colors